movies.json

```diff
@@ -1,802 +1,802 @@
 [
```

Comment:
This can be helpful in the dev process but shouldn't be committed, gotta keep GitHub clean.